features/account/registering/verifying_your_email_address.feature

```diff
@@ -0,0 +1,61 @@
+@account_verification
+Feature: Verifying account's email address
+    In order to take full advantage of my account
+    As a Visitor
+    I want to be able to verify my email address
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And there is user "ted@example.com" identified by "pswd123"
+        And "ted@example.com" account has not yet been verified
+
+    @todo
+    Scenario: Being not verified after registration
+        When I registered with email "evan@example.com" and password "drowssap"
+        Then I should be logged in
+        But my account should not be verified
+
+    @todo
+    Scenario: Verification email gets sent after registering
+        When I registered with email "evan@example.com" and password "drowssap"
+        Then a email should be sent to "evan@example.com"
+
+    @todo
+    Scenario: Getting verified after clicking the link in the verification message
+        When I verify my email address
+        Then I should be on homepage
+        And I should be notified that the verification was successful
+
+    @todo
+    Scenario: Being unable to log in without verifying my account
+        Given I am not logged in
+        When I try to log in with "ted@example.com" and "pswd123"
+        Then I should not be logged in
+        And I should be notified that the account needs to be verified
+
+    @todo
+    Scenario: Resending the verification email as a logged in user
+        Given I am logged in user
```

Comment:
And I want to resend my email address verification email
When I resend it
Then an email address verification email should be sent to "ted@example.com"